inc/spbc-settings.php

```diff
@@ -2747,7 +2746,8 @@ function spbc_list_table__get_args_by_type( $table_type ){
 					'sql' => array(
 //								'where' => ' WHERE severity IN("CRITICAL", "DANGER", "SUSPICIOUS") AND status <> "QUARANTINED"',
 						'where' => ' WHERE severity IN("CRITICAL") AND status <> "QUARANTINED"',
-					)
+					),
```

Comment:
As I see, you have added a sorting order for several categories. Please, tell why did you do this? Do we need this in this branch?

Reply:
I removed default order by and added orde by to every cases